Update 2020_q1.md (#733)
sourcegraph/about: company/okrs/2020_q1.md

   1. Provide a board-approved budget (including spending and hiring plans) to each department head. => Preliminary hiring and spend plan is done, but remains flexible in the face of COVID-19-related turbulence this year.
   1. Provide dashboards to each department head that are used on at least a weekly basis (prioritizing [sales](https://sourcegraph.looker.com/dashboards/123), [marketing](https://sourcegraph.looker.com/dashboards/118), and customer success). => Done for Marketing (created, and used on a weekly basis), created for Sales, draft for CE.

## People Ops

1. **People Ops: Build a consistent hiring process that supports candidates, interviewers, and hiring managers.**
   1. A scheduling coordinator is brought on to take over scheduling and candidate interfacing. ==> Done, scheduling coordinator started March 18.
   1. Each open role has its own customized hiring flow, clearly assigns owners to each stage, and is fully documented in     our interview repository. ==> In progress as of March 18.

>    1. Our interview repository documents interviewing best practices for all roles and instructions for interviewers. ==> Not done.

Reviewer: ```suggestion
   1. Our [interview repository](https://github.com/sourcegraph/interviews) documents interviewing best practices for all roles and instructions for interviewers. ==> Not done.
```

add link for convenience

> 1. **PeopleOps: Build an onboarding process that sets teammates up for success in a global, all-remote environment.**

Reviewer: ```suggestion
1. **People Ops: Build an onboarding process that sets teammates up for success in a global, all-remote environment.**
```

Reviewer: for consistency

   1. Each team has its own customized onboarding process, specific to the roles’ functions. ==> Not done.
   1. Our handbook describes updated, all-remote policies and offers resources for newly remote teammates, internationally      based teammates, etc. ==> Not done.

> 1. **PeopleOps: Create an inclusive people-ops operating system.**

Reviewer: ```suggestion
1. **People Ops: Create an inclusive people-ops operating system.**
```

Reviewer: for consistency
